browser/src/Services/SyntaxHighlighting/SyntaxHighlightingReducer.ts

```diff
@@ -134,7 +134,10 @@ export const linesReducer: Reducer<SyntaxHighlightLines> = (
                 const oldLine = updatedBufferState[i]
                 const newLine = action.lines[i]
 
-                if (oldLine && oldLine.line === newLine) {
+                // check if the buffer version has changed and if so
+                // update the line - rather than check if specific line
+                // is changed
+                if (oldLine && oldLine.version === action.version) {

```

Comment:
Does making this condition:

```
if (oldLine && oldLine.version >= action.version) {
```

not work? I'm thinking it may be a bit safer, as we likely don't need to re-apply updates for an old version.